entrypoint.sh

```diff
@@ -19,6 +20,14 @@ fi
 
 git fetch --unshallow
 
-for PULL_ID in $PULL_IDS; do
+if [ -z "${BASE_FOLDER}" ]; then
+  for PULL_ID in $PULL_IDS; do
   PRONTO_PULL_REQUEST_ID=$PULL_ID /usr/local/bundle/bin/pronto run -f github_status github_pr -c origin/master
-done
+  done
+else
+  cd ${BASE_FOLDER}
+  for PULL_ID in $PULL_IDS; do
+  PRONTO_PULL_REQUEST_ID=$PULL_ID /usr/local/bundle/bin/pronto run -f github_status github_pr -c origin/master
+  done
+fi
```

Comment:
Can't we simplify this? The commands seem to be the same, no?
```bash
if [ ! -z "${BASE_FOLDER}" ]; then
  cd ${BASE_FOLDER}
fi

for PULL_ID in $PULL_IDS; do
  PRONTO_PULL_REQUEST_ID=$PULL_ID /usr/local/bundle/bin/pronto run -f github_status github_pr -c origin/master
done
```

Reply:
totally agree